source/index.txt

```diff
@@ -68,6 +68,11 @@ binaries:
 Versioning
 ----------
 
+.. important::
+
+   It is recommended you always use the latest release version of the 
+   {+dbtools+}.
+
```

Comment:
Can we add a small code snippet here showing users how to check their current tool version?

I believe that checking any of the tool binaries should indicate the current version of the tools installed.

So something like:

```
To check your current version of the {+dbtools+}, run the following command on your terminal:

.. code-block:: shell

   mongodump --version

```

Reply:
Thanks Ian, it has been updated!